### ISSUE TITLE
ColorPicker tooltip for color name doesn't dismiss when scrolling away

### ISSUE THREAD

**Issue by asklar** [microsoft/microsoft-ui-xaml#5722]
**Describe the bug**

WinUI 2.6 UWP on Desktop

Add a ScrollViewer and inside it, a ColorPicker
select a color, a tooltip with the closest color name shows up
scroll the scrollviewer

Observe that the tooltip does not scroll with the ColorPicker as we scroll, and in fact even after the ColorPicker is scrolled off the screen, the tooltip remains.

**Comment by robloo** [comment]
Duplicate of https://github.com/microsoft/microsoft-ui-xaml/issues/3237

**Comment by StephenLPeters** [comment]
yup, unfortunately still hasn't been fixed :/

**Comment by robloo** [comment]
I thought a fix was proposed in the other issue but was waiting on the Microsoft team for feedback.
